flutter-studio/testSrc/io/flutter/tests/util/WizardUtils.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.tests.util;
+
+import com.android.tools.idea.tests.gui.framework.FlutterGuiTestRule;
+import com.android.tools.idea.tests.gui.framework.fixture.newProjectWizard.NewFlutterProjectWizardFixture;
+import io.flutter.module.FlutterProjectType;
+import org.jetbrains.annotations.NotNull;
+
+public class WizardUtils {
+  private WizardUtils() {
+  }
+
+  public static void createNewApplication(@NotNull FlutterGuiTestRule guiTest) {
+    createNewProject(guiTest, FlutterProjectType.APP, null, null, null, null, null);
```

Comment:
Perhaps overload `createNewProject()` so we don't need to pass as many `null`s in here?

Reply:
Do you mean creating a bunch of methods, each calling the next, and adding another null parameter? Or maybe just one more createNewProject() that takes two parameters? That would be used in these three methods. I think that makes sense.

Reply:
Yeah, likely just one overload of createNewProject(), that the three methods could call.